package.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh -e
+LAMBDAS=$1
+
+npx webpack --env.lambdas=${LAMBDAS}
+
+bundle_dir="build/bundle/"
+artifact_dir="build/artifacts/"
```

Comment:
Think the ops guys would like this dir at the root level

Reply:
Will catch up with @JamieKellyBJSS on this as he's doing a write-up on build process.

Reply:
Root level "artefacts/" please. Even if that's achieved by a copy/move at the end.

Reply:
Done.